### PR TITLE
chore: add eslint 

### PR DESCRIPTION
**Pull request by Krishks369**
Added eslint to template for better readability

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+node_modules
+test/temp
+template/src/api/routes.js
+template/src/api/services/$$channel$$.js
+template/src/lib/asyncapi.js
```
Comment (derberg) [inline .eslintignore:5]: shouldn't entire `template` dir be ignored?
also make sure to remove empty spaces please

**Review by derberg — CHANGES_REQUESTED** [review]
great job mate!
there is just one thing to improve

later if you have time, would love to get the same PR in https://github.com/asyncapi/nodejs-template 🙏🏼

File: .eslintignore (hunk 1 above)
Comment (Krishks369) [reply .eslintignore]: Done:+1:

**Comment by Krishks369** [comment]
> great job mate! there is just one thing to improve
> 
> later if you have time, would love to get the same PR in https://github.com/asyncapi/nodejs-template 🙏🏼

Sure, will do it

**Comment by derberg** [comment]
oh, sorry mate
you were actually right
now I know why you ignored only specific files
you ignored the templates, but not all files are templated, thus can be checked by eslint.

sorry about it 😞  just please bring back previous state of `.eslintignore` and only remove empty lines 🙏🏼

**Review by derberg — APPROVED** [review]
LGTM 🚀

**Pull request merged**